modules/tinymce/src/themes/silver/main/ts/Events.ts

```diff
@@ -49,6 +49,7 @@ const setup = (editor: Editor, mothership: Gui.GuiSystem, uiMothership: Gui.GuiS
   // Window events
   const onWindowScroll = (evt: UIEvent) => broadcastEvent(SystemEvents.windowScroll(), DomEvent.fromRawEvent(evt));
   const onWindowResize = (evt: UIEvent) => {
+    resizeMothership();
```

Comment:
This isn't the way to do this, we should listen to the event fired on the mothership instead. Take a look at setting up the events in `Render.ts` when the sink is created (given it's alloy, let me know if you need a hand).

- [ ] Listen to the silver event, instead of passing a callback via here.

Reply:
Worth noting if we do this using Alloy events, then we don't need the manual call in renderUi either, which should be a little cleaner.